portal/src/index.js

```diff
@@ -1,24 +1,24 @@
 import Vue from 'vue';
```

Comment:
Dit is op zich een prima file zo, veel meer Vue. Alles mag ook naar main.js wat mij betreft.

Reply:
Zal dat later nog wel doen (daar wil ik ook nog wat opschonen).